lib/tasks/metrics.rake

```diff
@@ -0,0 +1,26 @@
+require 'csv'
+require 'benchmark'
+
+namespace :metrics   do
+  desc 'Generating # of portfolios per organization'
+  task :portfolios => :environment do
```

Comment:
We want to create a per-tenant report that includes more than one metric at a time.  I would name this task `tenant`.